python/ray/data/dataset.py

```diff
@@ -3050,6 +3056,9 @@ def iter_torch_batches(
         collate_fn: Optional[
             Callable[[Union[np.ndarray, Dict[str, np.ndarray]]], Any]
         ] = None,
+        finalize_fn: Optional[
```

Comment:
I don't think we want to expose this in the API

Reply:
so users wouldn't be specifying their own custom `finalize_fn`? Would we always just use the default one?

Reply:
yep